src/picasso-def/components/labels.js

```diff
@@ -200,10 +201,19 @@ export const createOverlayLabel = ({ node, avgColor, width, height, renderer, ge
       fontSize,
       x,
       y,
-      fill: getContrastingColorTo(avgColor),
+      fill,
+      stroke: getContrastingColorTo(fill), // helium supports stroke
+      strokeWidth: 1,
       opacity: 0.7,
       baseline: 'central',
-      data: { ...node.data, depth: node.depth, overlay: true },
+      data:
+        level < 2
```

Comment:
why compare with 2 (it feels wrong to compare to a fixed number)

Reply:
The second level is where the overlays are done.

Reply:
Now using your suggestion below, it fixed the issue.